src/structures/Presence.js

```diff
@@ -92,6 +92,21 @@ class Activity {
     this.applicationID = data.application_id || null;
 
     /**
+     * Timestamps for the activity
+     * @type {Object}
```

Comment:
This is nullable.
And maybe add `@property` here to describe what it actually is.